nuxeo-notification-stream-core/src/main/java/org/nuxeo/ecm/platform/notification/processor/SubscriptionsProcessor.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * (C) Copyright 2018 Nuxeo SA (http://nuxeo.com/).
+ * This is unpublished proprietary source code of Nuxeo SA. All rights reserved.
+ * Notice of copyright on this source code does not indicate publication.
+ *
+ * Contributors:
+ *     Gildas Lefevre
+ */
+package org.nuxeo.ecm.platform.notification.processor;
+
+import static org.nuxeo.runtime.stream.StreamServiceImpl.DEFAULT_CODEC;
+
+import java.util.Collections;
+import java.util.Map;
+
+import org.nuxeo.ecm.platform.notification.NotificationStreamCallback;
+import org.nuxeo.ecm.platform.notification.NotificationStreamConfig;
+import org.nuxeo.ecm.platform.notification.message.SubscriptionAction;
+import org.nuxeo.lib.stream.codec.Codec;
+import org.nuxeo.lib.stream.computation.AbstractComputation;
+import org.nuxeo.lib.stream.computation.ComputationContext;
+import org.nuxeo.lib.stream.computation.Record;
+import org.nuxeo.lib.stream.computation.Topology;
+import org.nuxeo.runtime.api.Framework;
+import org.nuxeo.runtime.codec.CodecService;
+import org.nuxeo.runtime.stream.StreamProcessorTopology;
+
+/**
+ * StreamProcessor for saving the new subscriptions or unsubscribe users for a given notification.
+ *
+ * @since XXX
+ */
+public class SubscriptionsProcessor implements StreamProcessorTopology {
```

Comment:
this is not used anymore and can be removed, same for NotificationSettingsProcessor